config/locales/en.yml

```diff
@@ -8,3 +8,15 @@ en:
       processing: "Processing"
       succeeded: "Succeeded"
       failed: "Failed"
+    shared:
+      link:
+        back_to_home: "Back to home page"
+    details:
```

Comment:
It is best not to map translation keys specifically to pages (as these can change) but by domain. So the following attributes could still fit under `keywords`.

Reply:
Fixed in 9a6a72b1d07fc729f526a3e0f050536f7879ee2c